scripts/compare.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+
+import sys
+import getopt
+import shapely.geometry
+from svg.path import parse_path as parse_svg_path
+from svg.path import Path as SvgPath
+from svg.path import Move as SvgMove
+from svg.path import Line as SvgLine
+from svg.path import Close as SvgClose
+from svg.path import CubicBezier as SvgCubicBezier
+from svg.path import Arc as SvgArc
+from xml.dom import minidom
+
+stroke_width = 1.25
+
+#FIXME:
+width="2222.5mm"
+height="2470.1499mm"
+viewBox="0 0 2222.5 2470.1499"
+
+def get_svg_polygons(xml):
+	polygons = {}
+
+	# Process traces
+	elements = xml.getElementsByTagName("path")
+	for element in elements:
+		element_id = element.getAttribute('id')
+
+		print('Parsing SVG child elements:', element_id)
+
+		element_d = element.getAttribute('d')
+		element_d_segments = parse_svg_path(element_d)
+
+		nodes = []
+		for segment in element_d_segments:
+			#print(segment)
+			if isinstance(segment, SvgMove):
+				nodes.append(segment.end)
+			elif isinstance(segment, SvgLine) or isinstance(segment, SvgClose):
+				nodes.append(segment.end)
+			elif isinstance(segment, SvgCubicBezier):
+				#FIXME: Warn especially if it's very straight?
+				print("warning: %s uses bezier curve; should not be used!" % element_id)
+				steps = 5
+				for step in range(steps):
+					t = (step + 1) / steps
+					nodes.append(segment.point(t))
+			elif isinstance(segment, SvgArc):
+				print("Unsupported arc!")
+				assert(False)
+				nodes = []
+				break
+			else:
+				assert(False)
+
+		# Create a new SVG
+		if len(nodes) >= 2:
+
+			# Turn nodes into tuples
+			nodes = [(node.real, node.imag) for node in nodes]
+
+			line_string = shapely.geometry.LineString(nodes)
+			line_string_polygon = line_string.buffer(stroke_width/2.0, resolution=4)
```

Comment:
Also should hardcode the cap-style and corner-style, so that it doesn't break when shapely changes.